cli/server/server.go

```diff
@@ -107,11 +108,23 @@ func getConfigFromContext(ctx *cli.Context) (config.Config, error) {
 	return config.Load(configPath, net)
 }
 
-// handleLoggingParams enables debugging output is that's requested by the user.
-func handleLoggingParams(ctx *cli.Context) {
+// handleLoggingParams enables debugging output and logPath requested by the user .
+func handleLoggingParams(ctx *cli.Context) error {
 	if ctx.Bool("debug") {
 		log.SetLevel(log.DebugLevel)
 	}
+
+	if logPath := ctx.String("log-path"); logPath != "" {
+		if err := io.MakeDirAndFile(logPath, "logger"); err != nil {
```

Comment:
Not sure it's worth creating directories here.

Reply:
hm. Dunno. But we don't restrict user anyhow form setting log-path as "./blahblah/ololo.log"
Anyway we have to handle this case. Otherwise we got an error. So I think it's good to allow user to create dir. Why not if we can? =)